labs/fabric-healthcare-payer-prescription

```diff
@@ -0,0 +1,23 @@
+# Lab Name
+Modern Pharmacy Management
+
+# Short Description
+This project will focus on leveraging blockchain to transform and modernize prescriptions management by enabling flexibility and managing fraud.
+
+# Scope of Lab
+The scope of the lab is to develop a proof of concept (POC) for a blockchain-based presecription management solution that covers general use case in healthcare around prescriptions and its fullfilment.
+
+This repo will have a end user application and the blockchain built using hyperledger fabric. 
+
+In addition we plan to leverage Joget open source platform for building applications visually. 
+
+# Initial Committers
+Initial Commiters 
+- https://github.com/dewanz
+- https://github.com/kentydotcom
+- Anyone is welcome to join. 
+
+# Sponsor
+Richard Bloch richardbloch@businesslearninginc.com, chair of Healthcare SIG
+Raveesh Dewan rdewan@joget.com, chair of Healthcare Payer Sub Group
+- https://github.com/dewanz
```

Comment:
Template also contains a `Pre-existing repository` section. Please add this section and specify none if that is the case.